tests/test_edgeql_select.py

```diff
@@ -3797,6 +3797,33 @@ async def test_edgeql_select_empty_05(self):
                 } ORDER BY .number;
                 """)
 
+    async def test_edgeql_select_empty_object_01(self):
+        await self.assert_query_result(
+            r'''
+            WITH MODULE test
+            SELECT <Issue>{}
+            ''',
+            [],
+        )
```

Comment:
While it would be equivalent, can you also add a test for `SELECT count(<Issue>{}) = 0`?